go/signature/signature_key_templates.go

```diff
@@ -67,12 +113,22 @@ func createECDSAKeyTemplate(hashType commonpb.HashType, curve commonpb.EllipticC
 	return &tinkpb.KeyTemplate{
 		TypeUrl: ecdsaSignerTypeURL,
 		Value:   serializedFormat,
+		OutputPrefixType: prefixType,
 	}
 }
 
 // ED25519KeyTemplate is a KeyTemplate that generates a new ED25519 private key.
 func ED25519KeyTemplate() *tinkpb.KeyTemplate {
 	return &tinkpb.KeyTemplate{
 		TypeUrl: ed25519SignerTypeURL,
+		OutputPrefixType: tinkpb.OutputPrefixType_TINK,
 	}
 }
+
+// ED25519KeyWithoutPrefixTemplate is a KeyTemplate that generates a new ED25519 private key.
+func ED25519KeyWithoutPrefixTemplate() *tinkpb.KeyTemplate {
+	return &tinkpb.KeyTemplate{
+		TypeUrl: ed25519SignerTypeURL,
+		OutputPrefixType: tinkpb.OutputPrefixType_RAW,
+	}
+}
```

Comment:
Please add a newline at the end of this file.

Reply:
forgot to run `go fmt` on this file.. good catch.. I'll update

Reply:
done